doc/PaaS.md

```diff
@@ -0,0 +1,37 @@
+# Platform as a service
```

Comment:
does it make sens to have this PaaS.md document in the repository?

Reply:
Deleted .

Reply:
ok